config/_default/menus/menus.en.yaml

```diff
@@ -0,0 +1,1265 @@
+main:
+
+#####################
+## GETTING STARTED ##
+#####################
+
+  - name: "Getting Started"
+    identifier: "getting_started"
+    url: "getting_started/"
+    pre: "nav_start"
+    weight: 10000
+
+  - name: "Agent"
+    identifier: "getting_started_agent"
+    url: "getting_started/agent/"
+    parent: "getting_started"
+    weight: 1
+
+  - name: "Datadog"
+    url: "getting_started/application/"
+    parent: "getting_started"
+    weight: 2
+
+  ## GETTING STARTED - Integrations
+
+  - name: "Integrations"
+    identifier: "getting_started_integrations"
+    url: "getting_started/integrations/"
+    parent: "getting_started"
+    weight: 3
+  - name: "Prometheus/Openmetrics"
+    identifier: "getting_started_prometheus"
+    url: "getting_started/integrations/prometheus"
+    parent: "getting_started_integrations"
+    weight: 301
+
+  ## GETTING STARTED - Traces
+
+  - name: "Collect traces"
+    identifier: "getting_started_collect_traces"
+    url: "getting_started/tracing/"
+    parent: "getting_started"
+    weight: 4
+  - name: "Distributed Tracing"
+    identifier: "getting_started_distributed_tracing"
+    url: "getting_started/tracing/distributed-tracing/"
+    parent: "getting_started_collect_traces"
+    weight: 401
+
+  - name: "API"
+    identifier: "getting_started_api"
+    url: "getting_started/api/"
+    parent: "getting_started"
+    weight: 5
+
+  - name: "Learning Center"
+    url: "getting_started/learning_center/"
+    parent: "getting_started"
+    weight: 6
+
+#############
+## TAGGING ##
+#############
+
+  - name: "Tagging"
+    url: "tagging/"
+    pre: "nav_tagging"
+    identifier: "tagging_"
+    weight: 20000
+
+  - name: "Assigning tags"
+    url: "tagging/assigning_tags/"
+    parent: "tagging_"
+    weight: 1
+
+  - name: "Using tags"
+    url: "tagging/using_tags/"
+    parent: "tagging_"
+    weight: 2
+
+###########
+## AGENT ##
+###########
+
+  - name: "Agent"
+    url: "agent/"
+    pre: "nav_agent"
+    weight: 30000
+    identifier: "agent"
+
+  ## AGENT -  Basic Agent usage
+
+  - name: "Basic Agent Usage"
+    url: "agent/basic_agent_usage/"
+    parent: "agent"
+    identifier: "basic_agent_usage"
+    weight: 1
+  - name: "AIX"
+    identifier: "basic_agent_usage_aix"
+    url: "agent/basic_agent_usage/aix/"
+    weight: 101
+    parent: "basic_agent_usage"
+  - name: "Amazon Linux"
+    identifier: "basic_agent_usage_amazon_linux"
+    url: "agent/basic_agent_usage/amazonlinux/"
+    weight: 102
+    parent: "basic_agent_usage"
+  - name: "CentOS"
+    identifier: "basic_agent_usage_centos"
+    url: "agent/basic_agent_usage/centos/"
+    weight: 103
+    parent: "basic_agent_usage"
+  - name: "Debian"
+    identifier: "basic_agent_usage_deb"
+    url: "agent/basic_agent_usage/deb/"
+    weight: 104
+    parent: "basic_agent_usage"
```

Comment:
@ruthnaebeck @l0k0ms I spoke to Gus about this, he mentioned it was okay to remove `Docker` from the Basic Agent Usage section because it already appears below, so it was redundant.